src/components/student/exam/lastActions/Action.vue

```diff
@@ -0,0 +1,43 @@
+<template lang="html">
+  <div class="box">
+    <article class="media">
+      <div class="media-content">
+        <div class="content">
+          <p>
+            <strong>{{percentSolved}}% Solved</strong> <small>{{randomTime()}}</small>
```

Comment:
Similar to above, this is going to contain something like `timeLeft`, so it's better to rename it as such, and then only change the prop/method implementation.
(this is the way `percentSolved` was handled)